rules/aws_write_only_attributes.go

```diff
@@ -0,0 +1,127 @@
+package rules
+
+import (
+	"fmt"
+
+	"github.com/terraform-linters/tflint-plugin-sdk/hclext"
+	"github.com/terraform-linters/tflint-plugin-sdk/tflint"
+	"github.com/terraform-linters/tflint-ruleset-aws/project"
+	"github.com/zclconf/go-cty/cty"
+)
+
+// AwsWriteOnlyAttributesRule checks if a write-only attribute is available for sensitive input attributes
+// It emits a warning if the normal attribute is used, as that is a non-ephemeral attribute and the secret value is stored in state
+type AwsWriteOnlyAttributesRule struct {
+	tflint.DefaultRule
+
+	writeOnlyAttributes map[string]writeOnlyAttribute
+}
+
+type writeOnlyAttribute struct {
+	original             string
+	writeOnlyAlternative string
+	otherAlternative     string
```

Comment:
Leave this out, it is too big a change recommendation to offer without context. 

Adopting the write-only alternative to a sensitive attribute has clear effects that are documented in the rule:

1. Avoids inclusion in state
2. Prevents reference from other blocks

The `manage_*` attributes are a substantially different feature, opting the user into a new secret in Secrets Manager and notably _automatic rotation_. 

This deserves its own rule, even if both changes share the effect 1.